benefits/eligibility/forms.py

```diff
@@ -42,6 +42,29 @@ def clean(self):
             raise forms.ValidationError("reCAPTCHA failed")
 
 
+class InPersonEligibilityForm(forms.Form):
```

Comment:
Small note for code organization: let's have this in a file called `benefits/in_person/forms.py`

Reply:
Done here: https://github.com/cal-itp/benefits/pull/2317/commits/563b134f85bfbd34c56470aefe2abfa9e07bcc70